### PR TITLE
Fab 96 jira progress | Sprint 2

### PR DESCRIPTION
**Pull request by D22124491-NaveenMaheswaran**
Sprint 2 epic list
Sprint 2 retro, reviews and reports

### CODE REVIEW

File: Frontend-Angular/data-gpt-frontend/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    declarations: [AppComponent]
+  }));
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app).toBeTruthy();
+  });
+
+  it(`should have as title 'data-gpt-frontend'`, () => {
```
Comment (SeanMcCrossanTUD) [inline Frontend-Angular/data-gpt-frontend/src/app/app.component.spec.ts:17]: Should it still be called data-gpt-frontend? Maybe make it more configurable

**Review by SeanMcCrossanTUD — APPROVED** [review]
LG, would wait for @UmamaSumlinTasnuva to give it a more thorough review. I would change the name data-gpt though in the code

**Review by nikodemadamski — APPROVED** [review]
I can see you added a front end angular along side a json and blob storage as discussed before on call, it looks perfect by me, lets wait until umama also agrees with this code.

**Comment by SeanMcCrossanTUD** [comment]
It's also merging to main? I thought we were only merging into develop?

File: Frontend-Angular/data-gpt-frontend/src/app/app.component.spec.ts (hunk 1 above)
Comment (nikodemadamski) [reply Frontend-Angular/data-gpt-frontend/src/app/app.component.spec.ts:17]: This must be changed to develop instead of main asap

**Comment by nikodemadamski** [comment]
> It's also merging to main? I thought we were only merging into develop?

Yes this must be changed into develop instead of main asap

**Pull request merged**